packages/react/src/components/Tree/Tree.tsx

```diff
@@ -101,6 +101,8 @@ class Tree extends AutoControlledComponent<WithAsProp<TreeProps>, TreeState> {
 
   static autoControlledProps = ['activeItemIds']
 
+  static Item = TreeItem
```

Comment:
Why is this needed?

Reply:
I think I used in the screener test as selector then reverted that and left this. will remove, thanks!